src/playground-file-editor.ts

```diff
@@ -226,6 +226,11 @@ const mimeTypeToTypeEnum = (mimeType?: string) => {
     case 'text/javascript':
     case 'application/javascript':
       return 'js';
+    case 'text/jsx':
+      return 'jsx';
+    // tsx uses the same syntax highlighting as JSX
```

Comment:
This comment a bit confusing here. Belongs in the code-editor highlight  mode choosing function if anywhere.

Reply:
im just gonna remove it

Reply:
done!